gauntlet/src/lib.rs

```diff
@@ -380,7 +385,7 @@ pub async fn gauntlet(args: Args) -> Result<()> {
 
     println!("\nTotal analysis time: {total_time:?}");
 
-    if args.refresh {
+    if args.bless || args.update {
```

Comment:
Should this only fire on `--bless`

Reply:
No, we need to save the new diagnostics (and new hashes) back to the relevant config file when `--update` is passed. Otherwise `--update` would be a no-op that fails whenever something changed.